packages/smart/contracts/augur-para/FeePot.sol

```diff
@@ -33,9 +34,9 @@ contract FeePot is VariableSupplyToken, IFeePot {
         return string(abi.encodePacked("S_", reputationToken.symbol()));
     }
 
-    function depositFees(uint256 _amount) public returns (bool) {
+    function depositFees(uint256 _amount) override public returns (bool) {
         collateral.transferFrom(msg.sender, address(this), _amount);
-        uint256 _totalSupply = totalSupply; // after collateral.transferFrom to prevent reentrancy causing stale totalSupply
+        uint256 _totalSupply = totalSupply(); // after collateral.transferFrom to prevent reentrancy causing stale totalSupply
```

Comment:
I think you can still access the private variable `_totalSupply` in this derived contract, which would be cheaper.